woo-razorpay.php

```diff
@@ -1778,123 +1608,120 @@ function razorpay_webhook_init()
     $rzpWebhook->process();
 }
 
-define('RZP_PATH', plugin_dir_path( __FILE__ ));
+define('RZP_PATH', plugin_dir_path(__FILE__));
 define('RZP_CHECKOUTJS_URL', 'https://checkout.razorpay.com/v1/checkout.js');
 define('RZP_1CC_CSS_SCRIPT', 'RZP_1CC_CSS_SCRIPT');
 
-
 function enqueueScriptsFor1cc()
 {
     $siteurl = get_option('siteurl');
 
-    $domain = parse_url($siteurl, PHP_URL_HOST);
+    $domain    = parse_url($siteurl, PHP_URL_HOST);
     $domain_ip = gethostbyname($domain);
 
     //Consider https if site url is not localhost server.
-    if (filter_var($domain_ip, FILTER_VALIDATE_IP, FILTER_FLAG_IPV4 | FILTER_FLAG_NO_PRIV_RANGE | FILTER_FLAG_NO_RES_RANGE))
-    {
+    if (filter_var($domain_ip, FILTER_VALIDATE_IP, FILTER_FLAG_IPV4 | FILTER_FLAG_NO_PRIV_RANGE | FILTER_FLAG_NO_RES_RANGE)) {
         $siteurl = str_replace('http://', 'https://', $siteurl);
     }
 
     wp_register_script('1cc_razorpay_checkout', RZP_CHECKOUTJS_URL, null, null);
     wp_enqueue_script('1cc_razorpay_checkout');
 
-    wp_register_style(RZP_1CC_CSS_SCRIPT, plugin_dir_url(__FILE__)  . 'public/css/1cc-product-checkout.css', null, null);
+    wp_register_style(RZP_1CC_CSS_SCRIPT, plugin_dir_url(__FILE__) . 'public/css/1cc-product-checkout.css', null, null);
     wp_enqueue_style(RZP_1CC_CSS_SCRIPT);
 
-    wp_register_script('btn_1cc_checkout', plugin_dir_url(__FILE__)  . 'btn-1cc-checkout.js', null, null);
+    wp_register_script('btn_1cc_checkout', plugin_dir_url(__FILE__) . 'btn-1cc-checkout.js', null, null);
     wp_localize_script('btn_1cc_checkout', 'rzp1ccCheckoutData', array(
-      'nonce' => wp_create_nonce("wp_rest"),
-      'siteurl' => $siteurl,
-      'blogname' => get_bloginfo('name'),
-    ) );
+        'nonce'    => wp_create_nonce("wp_rest"),
+        'siteurl'  => $siteurl,
+        'blogname' => get_bloginfo('name'),
+    ));
     wp_enqueue_script('btn_1cc_checkout');
 }
 
 //To add 1CC button on cart page.
-add_action( 'woocommerce_proceed_to_checkout', 'addCheckoutButton');
+add_action('woocommerce_proceed_to_checkout', 'addCheckoutButton');
 
 function addCheckoutButton()
 {
-  add_action('wp_enqueue_scripts', 'enqueueScriptsFor1cc', 0);
+    add_action('wp_enqueue_scripts', 'enqueueScriptsFor1cc', 0);
 
-  if (isRazorpayPluginEnabled() && is1ccEnabled() )
```

Comment:
@poojarazor why you had removed this condition? Merchant has to enable the 1cc & plugin has to be activated right!! if you remove this condition then even if merchant has disabled the 1cc from wc dashboard, it will show magic checkout on cart page right!

Reply:
Haven’t removed from here because of version mismatch removed from here and added below